docs/cli.md

```diff
@@ -0,0 +1,507 @@
+  </td>
+  </tr>
+</table>
+
+### `cp`
+`cp` copies a file.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ cp new_name.txt stuff/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli016.png" class="invert-colors-in-dark-mode" alt="cp example"/>
+  </td>
+  </tr>
+</table>
+
+### `open` / `wslview`
+On macOS, `open` opens a file or directory with the default application, like a double click ([docs](https://ss64.com/osx/open.html)).
+
+On WSL (Windows), `wslview` opens a file or directory with the default application, like a double click ([docs](https://wslutiliti.es/wslu/man/wslview.html)).
+
+<video controls autoplay loop style="width: 100%; max-width: 640px; max-height: 480px;">
+  <source src="images/cli_vid002.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+<div class="primer-spec-callout warning" markdown="1">
+**WSL Pitfall:** You may need to install `wslu`, which includes `wslview`.
+```console
+$ sudo apt install wslu
+```
+</div>
+
+## Tips and Tricks
+
+### `clear` <kbd>Control</kbd> + <kbd>l</kbd>
+`clear` the terminal.  Pro-tip: <kbd>Control</kbd> + <kbd>l</kbd>.  That's a lowercase L.
+
+<video controls autoplay loop style="width: 100%; max-width: 640px; max-height: 480px;">
+  <source src="images/cli_vid003.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Tab complete <kbd>TAB</kbd>
+<kbd>TAB</kbd> autocompletes a file or directory name. It's faster than typing the whole thing.
+
+Type the first part of a filename, then press <kbd>TAB</kbd>.  Press again to show multiple completion options.
+```console
+$ cd ~/src/e  # Press TAB twice to see options
+eecs280/     eecs281/     eecs485/
+```
+
+<video controls autoplay loop style="width: 100%; max-width: 640px; max-height: 480px;">
+  <source src="images/cli_vid004.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Previous Command <kbd>⬆</kbd>
+<kbd>⬆</kbd> shows previous commands. For example, you might repeat a compile command while debugging.
+
+<video controls autoplay loop style="width: 100%; max-width: 640px; max-height: 480px;">
+  <source src="images/cli_vid005.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Colors and Customization
+Colorize the output of `ls`.
+
+<img src="images/cli021.png" class="invert-colors-in-dark-mode" width="768px" alt="styled terminal example"/>
+
+#### Windows/WSL and Linux
+Verify you're using the Bash shell, typical on WSL Ubuntu Linux.
+```console
+$ echo $0
+-bash
+```
+{: data-variant="no-line-numbers" }
+
+Edit your shell customization file.
+```console
+$ open ~/.bash_profile
+```
+{: data-variant="no-line-numbers" }
+
+Add this line.  Whenever you type `ls`, you'll actually get `ls -G`, which adds color.
+```bash
+alias ls='ls -G'
+```
+{: data-variant="no-line-numbers" data-title="~/.bash_profile" }
+
+Close your terminal and reopen it.  Test
+
+FIXME screenshot on macOS.
```

Comment:
Show the "new" screenshot again.

Reply:
Marking this as resolved and continuing conversation on more recent thread.